app/src/main/java/io/github/droidkaigi/confsched2018/data/api/GithubApi.kt

```diff
@@ -0,0 +1,16 @@
+package io.github.droidkaigi.confsched2018.data.api
+
+import io.github.droidkaigi.confsched2018.data.api.response.Contributor
+import io.reactivex.Single
+import retrofit2.http.GET
+import retrofit2.http.Path
+import retrofit2.http.Query
+
+interface GithubApi {
+    @GET("/repos/{owner}/{repo}/contributors")
+    fun getContributors(@Path("owner") owner: String, @Path("repo") repo: String,
+                                  @Query("per_page") perPage: Int): Single<List<Contributor>>
+    //Help wanted! We can load user's bio from this api. But this api can only load one user.
```

Comment:
I think it's not necessary to load bio one by one because there are so many contributors on this project.

Reply:
You’re right.
I'll delete it. 👍